### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by opiquem**
Solution

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,23 @@
...
-  // write code here
+  const basicDiscount = 50;
+  const alternativeDicount = 20;
+  const basicCost = 40;
+
+  let sum = 0;
+
+  for (let i = 1; i <= days; i++) {
+    sum += basicCost;
+  }
```
Comment (vladyslav-lapshyn) [inline src/calculateRentalCost.js:39]: mb you can just multiplication `days` on `basicCost` without `for`

**Review by vladyslav-lapshyn — APPROVED** [review]
GJ 🔥

File: src/calculateRentalCost.js (hunk 1 above)
Comment (Anna-Tkachenko) [reply src/calculateRentalCost.js:39]: Agree, check @vladyslav-lapshyn comment, we can avoid using loops here

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
⭐️

**Pull request closed without merging**